### ISSUE TITLE
Fully Release 0.7.2?

### ISSUE THREAD

**Issue by Jefferson-Faseler** [Michael-F-Bryan/mdbook-linkcheck#51]
Hey! Thanks so much for working on this!

TL;DR;
Would love to be able to simply install like all the other previous releases like so.
```
curl -L https://github.com/Michael-F-Bryan/mdbook-linkcheck/releases/download/v0.7.2/mdbook-linkcheck-v0.7.2-x86_64-unknown-linux-gnu.tar.gz -o - | tar -zxv
```

Curious if we can get 0.7.1 and 0.7.2 fully released with assets? Trying to use some of the latest features, but they won't work without some bugfixes. Building from source is not ideal for what I'm trying to do in CD and would be much easier if I could just install the binary in script for ubuntu. The alternatives require me to manage a docker image, or install rust as a dependency.

**Comment by Michael-F-Bryan** [comment]
Hmm... A release bundle should have been uploaded automatically by Travis. I'll have a look at CI in the morning and see why nothing happened.

**Comment by Jefferson-Faseler** [comment]
Seems the commit wasn't tagged? https://travis-ci.com/github/Michael-F-Bryan/mdbook-linkcheck/jobs/445561335#L921

I compared it to the [last successful unknown linux run](https://travis-ci.com/github/Michael-F-Bryan/mdbook-linkcheck/jobs/352918423) and the output was very different.

**Comment by Jefferson-Faseler** [comment]
Any updates on this?

**Comment by jhpratt** [comment]
Ping @Michael-F-Bryan

**Comment by Michael-F-Bryan** [comment]
I'm updating dependencies and switching from Travis/Appveyor to GitHub Actions for CI.

I'll be able to cut a v0.7.3 release when codespan `0.11.1` is released (see brendanzab/codespan#316).

**Comment by Michael-F-Bryan** [comment]
I still need to figure out why CI wasn't automatically attaching binaries to the release, but you should be able to [download them now](https://github.com/Michael-F-Bryan/mdbook-linkcheck/releases/tag/v0.7.4).
